index.html

```diff
@@ -0,0 +1,223 @@
+        <div>
+          <h2 class="tonic">Multi-Post
+            Stories</h2>
+          <div class="frame canopy">CANOPY</div>
+          <div class="frame counter"></div>
+          <div class="frame backend">Back End Dev</div>
+          <div class="frame counter"></div>
+          <div class="frame year">2015</div>
+        </div>
+        <p class="paragraph">A daily selection of privately personalized reads; 
+          no accounts or sign-ups required.</p>
+          <div class="tags">
+            <div class="tag html">html</div>
+            <div class="tag css">css</div>
+            <div class="tag javascript">javaScript</div>
+          </div>
+            <button class="projects"><span class="see-projects">see projects</span></button>
+      </section>
+    </article>
+    <section class="about">
+        <h2 class="myself">About
+          Myself</h2>
+          <p id="paragraph">
+            Hello I’m a software developer! I can help you build a product , 
+            feature or website Look through some of my work and experience! 
+            If you like what you see and have a project you need coded, 
+            don’t hestiate to contact me.
+          </p>
+          <div>
+            <div class="lets-connect">
+              <p class="connect">LET’S CONNECT</p>
+            </div>
+            <div class="media-icons">
+              <a href=""><img src="./images/github.png" alt=""></a>
+              <a href=""><img src="./images/Linkedin.png" alt=""></a>
+              <a href=""><img src="./images/peace.png" alt=""></a>
+                <a href=""><img src="./images/twitter.png" alt=""></a>
+                 <a href=""><img src="./images/mail.png" alt=""></a>
+            </div>
+            <button class="get"> <span class="resume">Get my resume</span></button>
+          </div>
+          <div class="languages">
+            <h5 class="lang">Languages</h5>
+            <img src="./images/Down.png" alt="">
+          </div>
+          <div id="lang-list" >
+            <ul>
+                <li class="list"><img src="./images/javascript.png" alt="" class="lang-image">
+                 <p class="lang-list">JavaSript</p></li>
+                <li class="list"><img src="./images/html.png" alt="" class="lang-image">
+                <p class="lang-list">HTML</p></li>
+                <li class="list" ><img src="./images/css.png" alt="" class="lang-image">
+                <p class="lang-list">CSS</p></li>
+            </ul>
+          </div>
+          <div class="frameworks">
+            <h5 class="skills">Frameworks</h5>
+            <img src="./images/Disabled.png" alt="">
+          </div>
+          <div class="line"></div>
+          <div class="frameworks">
+            <h5 class="skills">Skills</h5>
+            <img src="./images/Disabled.png" alt="">
+          </div>
+          <div class="line"></div>
+    </section>
+    <article>
+
+    </article>
```

Comment:
Kindly remove all the code you are not using, your site will look better and cleaner.